### DISCUSSION TITLE
Does anyone have a list of 3060 (not Ti) cards Amazon (US) links?

### DISCUSSION THREAD

**Discussion by lozbinks** [jef/streetmerchant#2374] [Help]
Does anyone have a list of 3060 (not Ti) Amazon (US) links or updated store TS file they would care to share please?

**Comment by lozbinks** [comment]
Update: I've done them now, if anyone wants them I can share.

**Reply by yilmazmidi** [reply]
Could you share it plse

**Reply by lozbinks** [reply]
`{
      brand: 'evga',
      cartUrl:
        'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08WM28PVH&Quantity.1=1',
      model: 'gaming',
      series: '3060',
      url: 'https://www.amazon.com/gp/product/B08WM28PVH',
    },
    {
      brand: 'gigabyte',
      cartUrl:
        'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08X4VJ6SB&Quantity.1=1',
      model: 'vision oc',
      series: '3060',
      url: 'https://www.amazon.com/gp/product/B08X4VJ6SB',
    },
    {
      brand: 'gigabyte',
      cartUrl:
        'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08X4V8N5Q&Quantity.1=1',
      model: 'gaming oc',
      series: '3060',
      url: 'https://www.amazon.com/gp/product/B08X4V8N5Q',
    },
    {
      brand: 'zotac',
      cartUrl:
        'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08W8DGK3X&Quantity.1=1',
      model: 'gaming',
      series: '3060',
      url: 'https://www.amazon.com/gp/product/B08W8DGK3X',
    },
    {
      brand: 'msi',
      cartUrl:
        'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08WPRMVWB&Quantity.1=1',
      model: 'ventus 2x oc',
      series: '3060',
      url: 'https://www.amazon.com/gp/product/B08WPRMVWB',
    },
    {
      brand: 'asus',
      cartUrl:
        'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08WHJPBFX&Quantity.1=1',
      model: 'tuf',
      series: '3060',
      url: 'https://www.amazon.com/gp/product/B08WHJPBFX',
    },
    {
      brand: 'msi',
      cartUrl:
        'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08WT47L8B&Quantity.1=1',
      model: 'gaming x trio',
      series: '3060',
      url: 'https://www.amazon.com/gp/product/B08WT47L8B',
    },
    {
      brand: 'msi',
      cartUrl:
        'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08WPJ5P4R&Quantity.1=1',
      model: 'gaming',
      series: '3060',
      url: 'https://www.amazon.com/gp/product/B08WPJ5P4R',
    },
    {
      brand: 'asus',
      cartUrl:
        'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08WGTL4CW&Quantity.1=1',
      model: 'strix oc',
      series: '3060',
      url: 'https://www.amazon.com/gp/product/B08WGTL4CW',
    },
    {
      brand: 'pny',
      cartUrl:
        'https://www.amazon.com/gp/aws/cart/add.html?ASIN.1=B08WRQ3JR1&Quantity.1=1',
      model: 'gaming',
      series: '3060',
      url: 'https://www.amazon.com/gp/product/B08WRQ3JR1',
    },`
